src/components/TeaserFront/Tile.js

```diff
@@ -125,9 +119,11 @@ const Tile = ({
       style={{ background }}
       className="tile"
     >
-      <div {...styles.imageContainer}>
-        <img src={image} alt="" {...styles.image} />
-      </div>
+      {image && (
+        <div {...styles.imageContainer}>
+          <img src={image} alt="" {...styles.image} />
```

Comment:
A general thing: Could we always forward and support alt props? Even if editors leave them empty often we should support them everywhere.